src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -25,16 +26,21 @@ public void setUp() {
         model = new ModelManager(getTypicalExpenseExpert(), new UserPrefs());
     }
 
-    @Test
-    public void execute_newExpense_success() {
-        Expense validExpense = new ExpenseBuilder().build();
-
-        Model expectedModel = new ModelManager(model.getExpenseExpert(), new UserPrefs());
-        expectedModel.addExpense(validExpense);
-
-        assertCommandSuccess(new AddCommand(validExpense), model,
-                String.format(AddCommand.MESSAGE_SUCCESS, validExpense), expectedModel);
-    }
+    // @Test
+    // public void execute_newExpense_success() {
+    //     Expense validExpense = new ExpenseBuilder().build();
+
+    //     Model expectedModel = new ModelManager(model.getExpenseExpert(), new UserPrefs());
+    //     expectedModel.addExpense(validExpense);
+    //     Budget newBudget = new Budget(model.getBudget().asInt() - validExpense.getAmount().asInt());
+
+    //     assertCommandSuccess(new AddCommand(validExpense),
+    //             model,
+    //             String.format(AddCommand.MESSAGE_SUCCESS, validExpense)
+    //             + "\n\n"
+    //             + String.format(AddCommand.BUDGET_EDITED, newBudget),
+    //             expectedModel);
+    // }
```

Comment:
Reason for test case being commented out?

Reply:
Uncommented. I tried to introduce more functionalities, have commented extra functionalities out.